docs/deployment.md

```diff
@@ -298,6 +299,21 @@ Examples
 
 Please add yours!
 
+#### Netlify Functions
+
+```js
+// functions/index.js
+const {
+  createLambdaFunction,
+  createProbot,
+} = require("@probot/adapter-aws-lambda-serverless");
+const appFn = require("../src/app");
+
+module.exports.webhooks = createLambdaFunction(appFn, {
+  probot: createProbot(),
+});
+```
```

Comment:
just making sure: did you confirm that this is working?

Could you maybe add a paragraph above the code example, explaining that Netlify is using `Lambda` as the backend for its functions, so the `@probot/adapter-aws-lambda-serverless` works for Netflify as well.